tests/test_imgtoolPy.py

```diff
@@ -0,0 +1,58 @@
+from imgtoolPy.imgtoolPy import shrink, sharpen, compress
+from numpy import random
+import pytest
+
```

Comment:
We might want to change this back to group_12 for now, since we might have merge conflicts later. 

```suggestion
from imgtoolPy.imgtoolPy import shrink, sharpen, compress
from numpy import random
import pytest

```

Reply:
Also, delete class() in test and delete repetitive tests.